app/views/index.html

```diff
@@ -26,44 +32,20 @@ <h1 class="nhsuk-heading-l">
 
 {% block content %}
   <div class="nhsuk-grid-row">
-    <div class="nhsuk-grid-column-full">
-
-      <ul class="nhsuk-grid-row nhsuk-card-group">
-        <li class="nhsuk-grid-column-one-third nhsuk-card-group__item">
-          {{ card({
-            "href": "/install",
-            "clickable": "true",
-            "heading": "Get started",
-            "headingClasses": "nhsuk-heading-m",
-            "description": "Download and installation guides"
-          }) }}
-        </li>
-        <li class="nhsuk-grid-column-one-third nhsuk-card-group__item">
-          {{ card({
-            "href": "/how-tos",
-            "clickable": "true",
-            "heading": "Guides",
-            "headingClasses": "nhsuk-heading-m",
-            "description": "Learn to create pages, use components, add data and more"
-          }) }}
-        </li>
-        <li class="nhsuk-grid-column-one-third nhsuk-card-group__item">
-          {{ card({
-            "href": "page-templates",
-            "clickable": "true",
-            "heading": "Page templates",
-            "headingClasses": "nhsuk-heading-m",
-            "description": "Copy and paste page templates into your project"
-          }) }}
-        </li>
-      </ul>
-
-      <hr class="nhsuk-u-margin-top-2 nhsuk-u-margin-bottom-6">
-
-      <div class="nhsuk-u-reading-width">
-        <p class="nhsuk-u-margin-bottom-0"><a href="https://service-manual.nhs.uk">Visit the NHS digital service manual</a> for guidance and examples.</p>
-      </div>
-
+    <div class="nhsuk-grid-column-one-third">
+      <h2 class="nhsuk-heading-m">Realistic</h2>
+      <p>Create a prototype that behaves like the real thing, across devices, using components and styles from the NHS Design System.
```

Comment:
Elsewhere we say ‘Use prototypes’ and ‘Test designs’, so the singular here stands out a bit.

```suggestion
      <p>Create prototypes that behave like the real thing, across devices, using components and styles from the NHS Design System.
```